ShowerShow/Startup.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.OpenApi.Models;
+using ShowerShow.DAL;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace ShowerShow
```

Comment:
Are we going to use program.cs or startup.cs? just to keep in mind the start point of the application.

Reply:
Program.cs. I have deleted startup.cs without asking anyone that is why it is added back. It requires consultation as it is indicated in the comments.

Reply:
It is there to salvage the stuff in case if we have to